browser/brave_app_controller_mac.mm

```diff
@@ -100,16 +102,35 @@ - (void)mainMenuCreated {
   _copyCleanLinkMenuItem = [editMenu itemWithTag:IDC_COPY_CLEAN_LINK];
   DCHECK(_copyCleanLinkMenuItem);
   [[_copyCleanLinkMenuItem menu] setDelegate:self];
+
+#if BUILDFLAG(ENABLE_TOR)
+  // Find and set up the main menu Tor item
+  NSMenu* fileMenu = [[[NSApp mainMenu] itemWithTag:IDC_FILE_MENU] submenu];
+  _torMainMenuItem = [fileMenu itemWithTag:IDC_NEW_OFFTHERECORD_WINDOW_TOR];
+  if (_torMainMenuItem) {
```

Comment:
Q. Can !_torMainMenuItem case happen? If it's not expected, I think having CHECK(torMainMenuItem) would be better than.

Reply:
Fixed in fff5a3152e5107ea62cf7b9b2ccf8997988522fe